sacc/sacc.py

```diff
@@ -208,11 +208,19 @@ def add_covariance(self, covariance):
         covariance: array or list
             2x2 numpy array containing the covariance of the added data points
             OR a list of blocks
+        overwrite: bool
+            If True, it overwrites the stored covariance matrix with the given
+            one.
 
         Returns
         -------
         None
         """
+        if self.has_covariance() and (overwrite is False):
```

Comment:
Should be `not overwrite` here instead of `overwrite is False` to allow for other non-true values like 0.

Reply:
You're right! Thanks!